### ISSUE TITLE
[BUG] LoginFaild 

### ISSUE THREAD

**Issue by YanCastle** [RFS-ADRENO/zca-js#47]
**Describe the bug**
Cookies IMEI UserAgent with EdgeExtendsion
1-3 day login success
it`s error after 3 days 
ErrorTip:  
The session key was improperly submitted or has reached its timeout. Direct the user to log in again to obtain another key

**To Reproduce**
1 Use 3-5 days cookie
2 do zalo.login()

![image](https://github.com/user-attachments/assets/8434e5a7-1e6d-4237-ba1d-27281de01bfc)

**Comment by RFS-ADRENO** [comment]
Did you try updating your cookies again?

**Comment by JustKemForFun** [comment]
expired cookies, so try get new cookies

load `https://chat.zalo.me/` check see account logout yet?
if account logouted, pls login again and get new cookies

becauce cookies have an expiration date, you need checking cookies daily
I can't leave cookies for too long because they're not secure

**Comment by YanCastle** [comment]
I am wondering if Zalo's web has the ability to automatically renew cookies

**Comment by RFS-ADRENO** [comment]
> I am wondering if Zalo's web has the ability to automatically renew cookies

Yes, in most cases, making a request updates the cookie. Therefore, if you're using cookies for login, ensure they are updated regularly.
